yeast-react-renderer/src/helpers/diff.ts

```diff
@@ -139,6 +139,16 @@ function renderNodeSegments(prop: string, modAssignment: ModificationAssignment,
 	return renderedSegments;
 }
 
+// escape pure whitespace segments
+function escapeHtmlText(s: string): string {
+	let escapedString = s;
+	if (s.trim() === '') {
+		escapedString = s.replace(/\s/g, '\u00A0');
```

Comment:
I'm not fully clear on what purpose this serves. Is this to get whitespace to show up in the diff since it will determine that `" " !== "\ u00A0"` and show it as changed? 

Can you add a comment to the code explaining why this function is necessary?